src/PKSim.Core/Services/SimulationConfigurationTask.cs

```diff
@@ -107,15 +108,20 @@ public SimulationConfiguration CreateFor(Simulation simulation, bool shouldValid
          //STEP8: Add Observers
          module.Add(_modelObserverQuery.AllObserversFor(module.Molecules, simulation));
 
-         //STEP9 once all building blocks have been created, we need to create the default parameter and molecule values values 
-         var initialConditions = _initialConditionsCreator.CreateFor(module, simulation);
-         module.Add(initialConditions);
+         //STEP9 once all building blocks have been created, we need to create the default parameter and compound molecule values
+         var allDefinedProteins = module.Molecules.Where(x => simulation.Individual.AllDefinedMolecules().AllNames().Contains(x.Name)).ToList();
```

Comment:
Feels like simulation.Individual.AllDefinedMolecules().AllNames() is going to be retrieved alk the time